lib/tools/system-calls.js

```diff
@@ -855,25 +845,43 @@ systemCallMethods.reboot = async function reboot (retries = DEFAULT_ADB_REBOOT_R
   });
 };
 
+/**
+ * @typedef {Object} rootResult
+ * @property {boolean} isRooted True if the device is rooted now
+ * @property {boolean} wasAlreadyRooted True if the device was already rooted
+ */
+
 /**
  * Switch adb server to root mode.
- *
- * @return {boolean} True of the switch was successful or false
- *                   if the switch failed.
+ * @param {boolean} unroot - Should we unroot instead of root? (default false)
+ * @return {boolean} True if the device is rooted.
  */
-systemCallMethods.root = async function root () {
+systemCallMethods.root = async function root (unroot = false) {
+  let cmd = unroot ? 'unroot' : 'root';
```

Comment:
const

Reply:
Should we add the `prefer-const` ESLint rule? I'm definitely in favor of _always_ using `const` unless you need to re-assign a variable.